docs/working-with-node/query-json-rpc.md

```diff
@@ -58,9 +62,22 @@ The command should return the following output:
 Now that we've confirmed the account we set up as premined has the correct balance, we can transfer some ether:
 
 ````bash
```

Comment:
This is JavaScript code, not ShellScript's

Reply:
Fixed 2be0d4a3eafabe597a9be201552ffc2aa869b870